app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_person = [Person(person["name"],
+                             person["age"]) for person in people]
```

Comment:
It would be better to format the list comprehension in a way to leave the `Person` instance initialization on one line, e.g.
```suggestion
    list_of_person = [Person(person["name"], person["age"]) 
                      for person in people]
```